tx_json.go

```diff
@@ -0,0 +1,457 @@
+package wavelet
```

Comment:
Rather than exposing a `Parser` insance, I believe what would be better is exporting a global function `ParseTransactionJSON(data []byte) (tag byte, payload []byte)`.